lib/helpers/pluralize.js

```diff
@@ -6,31 +6,36 @@ module.exports = pluralize;
  * Pluralization rules.
  */
 
+// David Xie: rebase pluralization rules from pluralize.js committed on Oct 6, 2021
+// License of pluralize.js: MIT License
```

Comment:
Given that Mongoose is Apache 2 licensed, based on my understanding we can't have part of the code being MIT... Is there any way we can simply just use a fixed version of pluralize instead?